migrate.py

```diff
@@ -610,5 +615,15 @@ def print_error(message):
     print_color(bcolors.FAIL, message)
 
 
+def name_clean(name):
+    newName = name.replace(" ", "_")
+    newName = re.sub(r"[^a-zA-Z0-9_]", "-", newName)
+
+    if (newName.lower() == "plugins"):
+        return newName + "-user"
```

Comment:
Is this a special case only you need or is this something general?
And why don't you simply return "plugins-user" ?

Reply:
`plugins` is a reserved repo name in Gitea and, I didn't want to hardcode the name part as then the if statement can just be extended if any more are found.

Reply:
As for the regex, its because Gitea only supports `[a-zA-Z0-9_-\.]` as a repo name but GitLab supports almost anything. (Added the missing chars)